packages/rest/src/coercion/coerce-parameter.ts

```diff
@@ -167,7 +168,7 @@ function coerceBoolean(data: string | object, spec: ParameterObject) {
 async function coerceObject(
   input: string | object,
   spec: ParameterObject,
-  options?: RequestBodyValidationOptions,
+  options: RequestBodyValidationOptions = defaultValidationOptions,
```

Comment:
I have mixed feeling about injecting `defaultValidationOptions` in so many places. Shouldn't we apply the default at the top level only? I think that means either in `parse-params.provider.ts` and `request-body.validator.ts`; or in `ajv-factory.provider.ts`.

@raymondfeng what's your opinion on this architectural aspect?

Reply:
I agree with @bajtos: let's try to apply the default options at the root of the call stack.

Reply:
It's now defined on `AjvFactoryProvider` as a static `readonly` property.